src/mvc/Application.js

```diff
@@ -261,6 +261,14 @@ define([
       if (arguments.length == 1) {
         return this._views[name];
       }
+
+      if (this._started) {
+        // @if DEBUG
+        blocks.debug.throwMessage('Views can not be constructed / added after the application has been started. Try to add your views sync.', __DEBUG_METHOD, 'Error');
```

Comment:
"Try to add yours views sync" - I don't get that. Can you make it a little more clear what should be done to fix the error.